src/components/Typography/Paragraph.tsx

```diff
@@ -1,4 +1,5 @@
-import { ParagraphVariants, SizeVariants, Typography } from '@/components/Typography'
+import { ParagraphVariants, SizeVariants } from './types'
+import { Typography } from './Typography'
```

Comment:
same as Header and Label

Reply:
There is a circular dependency issue while trying to use with the storybooks. Im not changing anything is still using the same Type, since the new is called `TypographyNew`